src/actions/importing/aioseo-posts-importing-action.php

```diff
@@ -366,7 +366,7 @@ public function map( $indexable, $aioseo_indexable ) {
 			}
 
 			if ( ! empty( $aioseo_indexable[ $aioseo_key ] ) ) {
-				$indexable->{$yoast_mapping['yoast_name']} = \call_user_func( [ $this, $yoast_mapping['transform_method'] ], $aioseo_indexable[ $aioseo_key ], $yoast_mapping );
+				$indexable->{$yoast_mapping['yoast_name']} = \call_user_func( [ $this, $yoast_mapping['transform_method'] ], $aioseo_indexable, $aioseo_key );
```

Comment:
Looking through the code we now call the transform method with 3 different sets of attributes in the 3 places it's potentially called.

This should be unified so that in all cases the same arguments are passed in the same order.